support/testing/address.go

```diff
@@ -44,3 +45,16 @@ func NewActorAddr(t *testing.T, data string) addr.Address {
 	}
 	return address
 }
+
+type rooter interface {
+	Root() (cid.Cid, error)
+}
+
+func MustRoot(t testing.TB, r rooter) cid.Cid {
+	t.Helper()
+	c, err := r.Root()
+	if err != nil {
+		t.Fatal(err)
+	}
+	return c
+}
```

Comment:
This is a weird file to put something about CIDs and storage in.

Reply:
mmm, yeah. That is a weird location...